src/function.rs

```diff
@@ -661,6 +662,61 @@ fn uuid(_context: Context) -> FunctionResult {
   Ok(uuid::Uuid::new_v4().to_string())
 }
 
+fn which(context: Context, s: &str) -> FunctionResult {
```

Comment:
Just checking: do you still want to do this, now that I'm using `Path::new(s)` instead of `PathBuf::from(s)`? I noticed your implementation does not handle `s.is_empty()` early either.